bin/describe/describe_pkg.ml

```diff
@@ -0,0 +1,41 @@
+open Import
+module Lock_dir = Dune_pkg.Lock_dir
+
+module Lock = struct
+  let term =
+    let+ lock_dir_paths =
+      let open Cmdliner.Arg in
+      value
+      & pos_all string []
+      & info
+          ~doc:
+            "The paths of the the lock directories to be inspected. Defaults to the lock \
+             directory specified in the context."
+          ~docv:"LOCKDIRS"
+          []
+    in
+    let lock_dir_paths =
+      match List.map lock_dir_paths ~f:Path.Source.of_string with
+      | [] -> [ Lock_dir.default_path ]
+      | lock_dir_paths -> lock_dir_paths
+    in
+    List.iteri lock_dir_paths ~f:(fun index lock_dir_path ->
```

Comment:
As a rule  you should use `Pp.enumerate` or `Pp.concat` to print a list of things.

Reply:
Sure, but this will require changing how we print solutions to the lock file. I purposefully avoided this to keep changes to a minimum.

Reply:
I don't see how using `Pp.concat_map` here will need changes anywhere else in the code.

Reply:
Sorry, I misunderstood the location.